backend/app/controllers/api/v1/accounts/users_controller.rb

```diff
@@ -22,6 +22,13 @@ def destroy
           head :ok
         end
 
+        def transfer_ownership
+          user = @users.find params[:user_id]
```

Comment:
`load_and_authorize_resource` makes things not obvious at first glance.

Reply:
Sry, I personally like magic behind `load_and_authorize_resource`, but maybe It would be more transparent to use `user = authorize! User.find(params[:user_id])` here. Hmm, maybe I will change it this way...